### PR TITLE
Added incrementinator 2.0 project 

### PR DESCRIPTION
**Pull request by prat1k-gupta**
### ✅ Check List (Check all the applicable boxes) 

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots
image

### CODE REVIEW

**Comment by prat1k-gupta** [comment]
completed the issue #186 under GSSOC 22

**Comment by NOiR-07** [comment]
Don't edit card class in CSS, it will affect other cards too. Instead of that you can make a separate class

File: index.html [hunk 1]
```diff
@@ -263,7 +263,19 @@ <h5 class="card-title">Rock Paper Scissor</h5>
...
+  <img class="card-img-top" src="assets/Images/incrementinator.png" alt="Card restaurant">
+  <div class="card-body">
+    <h5 class="card-title">Incrementinator 2.0</h5>
+    <p>Beginners project using html,css,js!!</p>
+    <a href="Public/incrementinator.html" class="btn btn-outline-primary">Try now</a>
+  </div>
+</div>
+<!-- end of incrementinator 2.0 -->
     </div> 
+
```
Comment (NOiR-07) [inline index.html:278]: remove the line

File: style.css [hunk 2]
```diff
@@ -38,6 +38,7 @@
 .card p{
     font-family: 'Architects Daughter';
     font-size: 18px;
+    color: black;
```
Comment (NOiR-07) [inline style.css:41]: remove this line too

**Comment by prat1k-gupta** [comment]
can you check it now

**Comment by NOiR-07** [comment]
delete the icc.png img

**Comment by prat1k-gupta** [comment]
deleted icc.png

**Pull request merged**